examples/fabric-bridge-app/linux/include/RpcClient.h

```diff
@@ -0,0 +1,27 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <platform/CHIPDeviceLayer.h>
+
+constexpr uint16_t kFabricAdminServerPort = 33001;
+
+CHIP_ERROR InitRpcClient(uint16_t rpcServerPort);
+
+CHIP_ERROR OpenCommissioningWindow(chip::NodeId nodeId);
```

Comment:
Probably good to document the return value here? Because it looks like to me we can get OK, even the response back is a failure.